docs/source/content/overview/linear_SVM.rst

```diff
@@ -72,16 +72,32 @@ separable data**. Here is an example of that kind of data.
 
 .. figure:: _img/SVM_Kernal.png
    :scale: 50%
-   :alt: Outliers
+   :alt: Kernel
 
    Ref:  https://www.analyticsvidhya.com/blog/2017/09/understaing-support-vector-machine-example-code/
 
 
-There's is no clear way to separate the stars from the circles. SVMs will be able to classify non-linearly separable
-data by using a trick called the **kernel trick**. Basically, what goes behind the scenes, is that it takes the points
-to a higher dimension, which turns non-linearly separable date to linear separable data. So the above figure would be
+There is no clear way to separate the stars from the circles. SVMs will be able to classify non-linearly separable
+data by using a trick called the **kernel trick**. Basically, the kernel trick takes the points
+to a higher dimension to turn non-linearly separable data to linear separable data. So the above figure would be
 classified with a circle that separates the data. 
 
+Here is an example of the kernel trick.
+
+.. figure:: _img/SVM_Kernel1.png
+   :scale: 50%
+   :alt: Kernel X Z graph
+
+   Ref:  https://www.analyticsvidhya.com/blog/2017/09/understaing-support-vector-machine-example-code/
+
+.. figure:: _img/SVM_Kernel2.png
```

Comment:
This image doesn't seem to be scaled the same as the others. Can you copy it directly from the ref and make sure it's the same size?